cms/djangoapps/contentstore/signals.py

```diff
@@ -28,3 +29,12 @@ def listen_for_library_update(sender, library_key, **kwargs):  # pylint: disable
     from .tasks import update_library_index
     if LibrarySearchIndexer.indexing_is_enabled():
         update_library_index.delay(unicode(library_key), datetime.now(UTC).isoformat())
+
+
+@receiver(SignalHandler.course_published)
+def remove_ora1_deprecation_cache(sender, course_key, **kwargs):  # pylint: disable=unused-argument
+    """
+    Receives signal and remove ora1 deprecated components cache
+    """
+    cache_key = 'ora1.components.{course}'.format(course=course_key)
+    cache.delete(cache_key)  # pylint: disable=maybe-no-member
```

Comment:
@ormsbee Here we remove the ORA1 components for a particular course upon `course_published` event. In this way we don't need to check the `modified` datetime value from `CourseStructure`